servers/Azure.Mcp.Server/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The Azure MCP Server updates automatically by default whenever a new release com
 ## 0.5.13 (Unreleased)
 
 ### Features Added
+- Added support for retrieving administration settings (currently purge protection state and soft delete retention days) of an Azure Key Vault via the command `azmcp-keyvault-admin-settings-get`.
```

Comment:
I don't think we need to advertise which settings are supported.

Reply:
I'd also avoid talking about it as "Key Vault administration settings". I think "Key Vault settings" is just fine.